src/solvers/dgsem_tree/dg_2d_subcell_limiters.jl

```diff
@@ -0,0 +1,193 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+function create_cache(mesh::TreeMesh{2}, equations,
+                      volume_integral::VolumeIntegralSubcellLimiting, dg::DG, uEltype)
+    cache = create_cache(mesh, equations,
+                         VolumeIntegralPureLGLFiniteVolume(volume_integral.volume_flux_fv),
+                         dg, uEltype)
+
+    A3dp1_x = Array{uEltype, 3}
+    A3dp1_y = Array{uEltype, 3}
+    A3d = Array{uEltype, 3}
+
+    fhat1_threaded = A3dp1_x[A3dp1_x(undef, nvariables(equations), nnodes(dg) + 1,
+                                     nnodes(dg)) for _ in 1:Threads.nthreads()]
+    fhat2_threaded = A3dp1_y[A3dp1_y(undef, nvariables(equations), nnodes(dg),
+                                     nnodes(dg) + 1) for _ in 1:Threads.nthreads()]
+    flux_temp_threaded = A3d[A3d(undef, nvariables(equations), nnodes(dg), nnodes(dg))
+                             for _ in 1:Threads.nthreads()]
+
+    container_antidiffusive_flux = Trixi.ContainerAntidiffusiveFlux2D{uEltype}(0,
```

Comment:
I am not a big fan of encoding the type in the variable name. It's a little bit like writing `a_int = 1`. Thus I suggest to drop the `container_` prefix and just call it `antidiffusive_fluxes` or something similar

Reply:
I agree with you there. I'll change it.
Then, we will have lines like `@unpack antidiffusive_flux1, antidiffusive_flux2 = cache.antidiffusive_fluxes`.
I really thought it would look worse:)